assets/blocks/reader-registration/style.scss

```diff
@@ -82,19 +95,35 @@
 	}
 
 	&__have-account {
-		font-size: 0.8125em;
-		line-height: 1.2307692308;
-		margin: 0.8rem 0;
+		align-items: center;
+		background: var( --newspack-secondary-color );
+		color: var( --newspack-secondary-contrast-color );
+		display: flex;
+		font-size: 1em;
+		justify-content: center;
+		left: 0;
+		line-height: 1.5;
+		min-height: 36px;
+		padding: 0 16px;
+		position: absolute;
```

Comment:
If the Registration block is placed on a page (not in a prompt), this makes the banner stick to the top of the viewport.

Reply:
I completely reworked this in https://github.com/Automattic/newspack-plugin/pull/1967/commits/207127595e4426fea51c90cdc76c8bc5f0951fa7

- I didn't like having a "floating" banner at the top. Instead, I moved the messaging next to the block title. So it's similar to the Sign In modal.